Changes to the CSS page. links, wordsmithing and html code highlighting

garster opened the pull request.

garster committed:
Code highlight some elements on css page. Fixes #210

garster committed:
Minor wordsmithing

garster committed:
More code highlighting fixes. Should close #210

garster commented:
@loganfranken I think I got it all.

loganfranken commented:
Hey @garster! Thanks for your work on this!

The only thing is it looks like some of the bad practice examples got corrected to be good practice examples?

garster commented:
@loganfranken do you mean when I changed the `.red` color from #CC0000 to #C00? 

I changed it because both the good and bad examples had the long-form. The example was about the name of the class. I feel the rest of the content which is only there to fill out the example, good or bad, should be good form. Many times I have copied some code like this and got bad stuff.

loganfranken commented:
Ahh gotcha, my bad: I see you corrected it on the bad practice example that was focused on a different concept. Nevermind! Thanks for humoring me!

dsaludares commented on css.md:
```diff
@@ -12,19 +12,18 @@ More specifically, *websites should not convey information by color alone*.
+CSS techniques and technologies are evolving rapidly. Website developers should
+try to understand the tradeoffs involved with their choices (loading one vs. many
+stylesheets, performance vs. organization, basic CSS vs. minified/compressed/compiled
+CSS). Website developers may want to consider using stylesheet minifies, compilers,
```
Change "minifies" to "minifiers"

dsaludares replied:
Change "minifies" to "minifiers"

dsaludares reviewed (changes requested):
Found one typo

garster committed:
typo minifies => minifiers
